src/style.css

```diff
@@ -2,0 +1,74 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src:
+    url(../src/fonts/Avenir-Book.ttf) format('truetype'),
+    url(../src/fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+.body {
+  margin: 0;
+  box-sizing: border-box;
+  padding-top: 20px;
+  font-family: Avenir, Arial, sans-serif;
+}
+
+.input {
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+  position: relative;
+  transition: box-shadow 0.3s;
+}
+
+.input__line::before {
+  content: '';
+  position: absolute;
+  background-image: url(/src/images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  transform-box: content-box;
+}
+
+.input__line--big::before {
+  width: 20px;
+  height: 20px;
+  top: 25px;
+  left: 26px;
+}
+
+.input__line--small::before {
+  width: 12px;
+  height: 12px;
+  top: 15px;
+  left: 13px;
+}
+
+.input:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+  transition: box-shadow 0.3s;
+}
+
+.input:focus {
+  box-shadow: 0 4px 4px 0 #00000040;
+  transition: box-shadow 0.3s;
+}
+
+.input__big {
```

Comment:
The `.input__big` class should not fix the height of the input (line 59). Let the content size dictate the height to avoid overflow or accidental scroll bars.